widget/tests/test_Toaster.html

```diff
@@ -7,60 +6,66 @@
 		@import "../../../dojo/resources/dojo.css";
 		@import "../../../dijit/themes/tundra/tundra.css";
 		@import "../../../dijit/themes/dijit.css";
-		@import "../../../dijit/tests/css/dijitTests.css"; 
+		@import "../../../dijit/tests/css/dijitTests.css";
 		@import "../Toaster/Toaster.css";
 	</style>
 
-	<script type="text/javascript" src="../../../dojo/dojo.js" data-dojo-config="isDebug:true, parseOnLoad: true"></script>
-	<script type="text/javascript">
-		dojo.require("dojox.widget.Toaster");
-		dojo.require("dojo.parser");	// scan page for widgets and instantiate them
+	<script src="../../../dojo/dojo.js" data-dojo-config="isDebug:true, async: 1"></script>
+	<script>
+		require([
+			"dojo/parser",
+			"dojo/topic",
+			"dojo/on",
+			"dojo/dom",
+			"dijit/registry",
+			"dijit/Tooltip",
+			"dojox/widget/Toaster",
+			"dojo/domReady!"
+		], function (parser, topic, on, dom, registry) {
+			parser.parse();
+			var toast = registry.byId("toast");
+			function showTestMessage(){
+				topic.publish("testMessageTopic",
+					"This is a message! It's kind of long to show message wrapping."
+				);
+			}
+			function showAnotherMessage(){
+				topic.publish("testMessageTopic",
+					{
+						message: "This is another message!",
+						type: "warning",
+						duration: 500
+					}
+				);
+			}
+			function showYetAnotherMessage(){
+				topic.publish("testMessageTopic",
+					{ message: "This is yet another message!" }
+				);
+			}
 
-		dojo.require("dijit.Tooltip");
-		
-		var toast = null;
-		function showTestMessage(){
-			dojo.publish("testMessageTopic", 
-				[ "This is a message! It's kind of long to show message wrapping."]
-			);
-		}
-		function showAnotherMessage(){
-			dojo.publish("testMessageTopic", 
-				[{
-					message: "This is another message!", 
-					type: "warning", 
-					duration: 500
-				}]
-			);
-		}
-		function showYetAnotherMessage(){
-			dojo.publish("testMessageTopic", 
-				[{ message: "This is yet another message!" }]
-			);
-		}
-
-		dojo.addOnLoad(function(){
-			toast = dijit.byId("toast");
+			on(dom.byId("submitButton1"), "click", showTestMessage);
+			on(dom.byId("submitButton2"), "click", showAnotherMessage);
+			on(dom.byId("submitButton3"), "click", showYetAnotherMessage);
```

Comment:
No need for `showToastMessage` etc. variables, the function definitions can be inlined here.